chatcommunicate.py

```diff
@@ -139,6 +139,9 @@ def on_msg(msg, client, room):
             if room_data.last_report_data != ():
                 _last_messages.reports[message.id] = room_data.last_report_data
 
+                if len(_last_messages.reports) > 50:
```

Comment:
You might want to change this to a `while` to keep the count down if multiple reports accidentally get added, sending the total above 51.

Reply:
@j-f1 This would only happen if another thread added to it. I'll add a lock instead